modules/extensions-spring-support/src/test/java/org/mule/test/module/extension/reconnection/ReconnectionTestCase.java

```diff
@@ -153,14 +159,22 @@ public void getRetryPolicyTemplateFromConfig() throws Exception {
     RetryPolicyTemplate template = (RetryPolicyTemplate) flowRunner("getReconnectionFromConfig").run()
         .getMessage().getPayload().getValue();
 
-    assertRetryTemplate(template, false, 3, 1000);
+    assertRetryTemplate(template, true, 3, 1000);
   }
 
   @Test
   public void getInlineRetryPolicyTemplate() throws Exception {
     RetryPolicyTemplate template = (RetryPolicyTemplate) flowRunner("getInlineReconnection").run()
         .getMessage().getPayload().getValue();
 
+    assertRetryTemplate(template, true, 30, 50);
```

Comment:
why are you asserting the the retry template is async here when you explicitly defined it as blocking?

Reply:
This corresponds to getInlineReconnection

    <flow name="getInlineReconnection">
        <reconnection:get-retry-policy-template config-ref="reconnection_Config">
            <reconnect count="30" frequency="50" blocking="false"/>
        </reconnection:get-retry-policy-template>
    </flow>